spark/src/main/java/org/apache/iceberg/actions/BaseSparkAction.java

```diff
@@ -64,6 +70,28 @@ protected JavaSparkContext sparkContext() {
     return sparkContext;
   }
 
+  protected abstract R doExecute();
```

Comment:
I don't think we need `doExecute` and `jobGroup` per discussion in [this](https://github.com/apache/iceberg/pull/2287/files#r589880210) thread.